### DISCUSSION TITLE
Change config to set up permissions differently

### DISCUSSION THREAD

**Discussion by celticpixie** [GriefPrevention/GriefPrevention#1823] [Ideas]
I have a minor issue on a server running griefprevention 16.17.1 and was wondering if there was any way you could implement a fix for a permissions issue.

The way our server runs with staff permissions is that we allow lower ranking staff to use /ignoreclaims but not /abandonclaims (/deleteclaims as per permissions list) - however - when staff type /ignoreclaims they are then also able to use /abandonclaims. Looking at the current config - there seems to be no way around this without taking away the ability for them to use /ignoreclaims.

Any way you could add this as a feature in the config file ? Saves us having to revoke the ignoreclaims permission in the first place :)

Thank you

**Comment by Jikoo** [comment]
Update GP. This was a bug that has been fixed.
